tests/Feature/Dashboard/BasicTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Tests\Feature\Dashboard;
+
+use DB;
+use Hash;
+use Mockery;
+use Tests\TestCase;
+
+class BasicTest extends TestCase {
+    public function testPageLoads() {
+        // Test the dashboard page loads.  Most of the work is done inside Vue, so a basic test is just that the
+        // Vue component exists.
+        $this->loginAsTestUser();
+        $response = $this->get('/dashboard');
+        $content = $response->getContent();
+
+        $this->assertNotFalse(strpos($content, '<DashboardPage'));
```

Comment:
Think you could use assertSee here if you wanted (https://laravel.com/docs/5.6/http-tests#assert-see)

Reply:
Will do.